### DISCUSSION TITLE
files extension work with image

### DISCUSSION THREAD

**Discussion by nico-tome** [TurboWarp/extensions#463] [Ideas]
Maybe add a feature to the "files" extensions that can read the pixels from a picture and give back them in a list.

**Comment by LilyMakesThings** [comment]
I think files extension is only meant to provide fundamental file utilities, such as uploading and downloading. I think this use case is a bit too esoteric for the extension.

**Reply by nico-tome** [reply]
Yes but it's can be really useful for example for import crosshair or profile picture
